packages/auth/.gitignore

```diff
@@ -0,0 +1,3 @@
+/express
+/next
+/next-auth
```

Comment:
these are checked in on purpose

Reply:
if these are not checked in preconstruct will complain about it during every build